### ISSUE TITLE
training fully convolution networks

### ISSUE THREAD

**Issue by rosenfeldamir** [vlfeat/matconvnet#285]
Hi, thanks for adding the fully convolutional networks for semantic segmentation!
Is it currently possible to train / finetune these models using matconvnet? i.e, what would be the appropriate loss function to use ?
Thanks

**Comment by vedaldi** [comment]
Yep, straight from the oven:

https://github.com/vlfeat/matconvnet-fcn

**Comment by rosenfeldamir** [comment]
Excellent ,thanks!!
-Amir
